### PR TITLE
delete group functionality and displaying completed labs

### PR DESCRIPTION
**Pull request by Mster115**
- [ ] No discrepancies across browsers. Eg. chrome vs safari
- [ ] Pages pass W3C Validation for HTML (https://validator.w3.org)
- [ ] CSS (https://jigsaw.w3.org/css-validator/)
- [x] Google sign-in is operational
- [ ] Accessibility functions
- [ ] Pages can scale without distorting page
- [ ] No dead links
- [ ] Navbar is consistent across the site
- [ ] Pages are screen-reader accessible
- [ ] Contrast meets standards for accessibility
- [ ] Pages are keyboard accessible
- [ ] Passes WAVE Evaluation Tool chrome extension

### CODE REVIEW

File: client/src/components/body/profilepage/GroupDetails.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const GroupDetails = (props) => {
                     setEnrolledStudents(data);
                 })
             }
+
```
Comment (AndreasLc1103) [inline client/src/components/body/profilepage/GroupDetails.js]: Extra new line. must've been done by the linter.

File: client/src/components/body/profilepage/components/ResultLimiter.js [hunk 2]
```diff
@@ -50,25 +51,26 @@ const ResultLimiter = (props) =>{
                                         <p className="grey-text">Enrolled on {student.enrolledDate.split("T")[0]}</p>
                                     </>
                                 </td>
-                                <td className="assigned-labs">
-                                    {assignedLabs.map((lab, index) => (
+                                    <LabCompletionBubbles studentid={student.userID} groupid={groupid}/>
+                                    {/* {assignedLabs.map((lab, index) => (
```
Comment (AndreasLc1103) [inline client/src/components/body/profilepage/components/ResultLimiter.js]: remove commented code

**Review by AndreasLc1103 — COMMENTED** [review]
Little minor changes, did not test for unit functionality but I can assure you that everything looks like its functionally correct.

**Comment by AndreasLc1103** [comment]
This review has been reviewed by @Marrufof and @AndreasLc1103

**Comment by Mster115** [comment]
> Little minor changes, did not test for unit functionality but I can assure you that everything looks like its functionally correct.

Sounds good, I'm gonna throw one more edge case at it tomorrow

File: client/src/components/exercise/lab4/pages/FormHintAccessible.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ class FormHintAccessible extends Component {
                 <ExtraNav />
 
                 <AppInstructions
-                    class={this.state.class}
+                    className={this.state.class}
```
Comment (jajibhee) [inline client/src/components/exercise/lab4/pages/FormHintAccessible.js:39]: Do you want to maybe destructure your class state:
const { class } = this.state

That way you can just pass it in as class instead of typing this.state.class over and over again

File: client/src/components/exercise/lab4/pages/FormHintInaccessible.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ class FormHintInaccessible extends Component {
                 </a>
                 <ExtraNav />
                 <AppInstructions
-                    class={this.state.class}
+                    className={this.state.class}
```
Comment (jajibhee) [inline client/src/components/exercise/lab4/pages/FormHintInaccessible.js:39]: Do you want to maybe destructure your class state:
const { class } = this.state

That way you can just pass it in as class instead of typing this.state.class over and over again

File: client/src/components/exercise/lab4/pages/FormSkipToMainFixed.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ class FormSkipToMainFixed extends Component {
 
                 <ExtraNav />
                 <AppInstructions
-                    class={this.state.class}
+                    className={this.state.class}
```
Comment (jajibhee) [inline client/src/components/exercise/lab4/pages/FormSkipToMainFixed.js:37]: Do you want to maybe destructure your class state:
const { class } = this.state

That way you can just pass it in as class instead of typing this.state.class over and over again

File: client/src/components/exercise/lab4/pages/FormSkipToMainBroken.js [hunk 6]
```diff
@@ -31,7 +31,7 @@ class FormSkipToMainBroken extends Component {
             <Fragment>
                 <ExtraNav />
                 <AppInstructions
-                    class={this.state.class}
+                    className={this.state.class}
```
Comment (jajibhee) [inline client/src/components/exercise/lab4/pages/FormSkipToMainBroken.js:34]: Do you want to maybe destructure your class state:
const { class } = this.state

That way you can just pass it in as class instead of typing this.state.class over and over again
